javascript-primitives-and-variables/main.js

```diff
@@ -0,0 +1,18 @@
+var fullName = 'Kenny Park';
+var isCool = true;
+var totalPets = 1;
+var unknown = null;
+var never =
```

Comment:
This is a syntax error. You shouldn't have an assignment operator not followed by a value. You can do this correctly like so: `var never;`